ash/src/vk/definitions.rs

```diff
@@ -1263,6 +1263,10 @@ impl ::core::default::Default for DeviceQueueCreateInfo<'_> {
 unsafe impl<'a> TaggedStructure for DeviceQueueCreateInfo<'a> {
     const STRUCTURE_TYPE: StructureType = StructureType::DEVICE_QUEUE_CREATE_INFO;
 }
+#[diagnostic::on_unimplemented(
+    message = "`{Self}` is not allowed in the `pNext` chain of `vk::DeviceQueueCreateInfo`",
+    label = "This argument does not extend `vk::DeviceQueueCreateInfo`"
```

Comment:
Shouldn't it be "`{Self}` does not extend `vk::DeviceQueueCreateInfo`" ?

Where does "This argument" come from ?

Reply:
That's funny, I had already replaced it locally with that while testing the links, exactly because it does not always need to be used in argument position.